src/app/models/isolator.model.ts

```diff
@@ -0,0 +1,33 @@
+export interface Isolator {
```

Comment:
Looks good but there is a duplication with /src/isolator.model.ts . Yours is in a better path though. Can we remove the old one?